config/locales/en/privacy_policy.yml

```diff
@@ -2,107 +2,100 @@ en:
+          How we check that the information we hold is accurate and up-to-date; and
+          How to make a complaint.
+        info_1: For more information about the above issues, please contact the;
+        address:
           title: The Data Protection Officer
-          email_address_html: <a href="mailto:privacy@justice.gov.uk">privacy@justice.gov.uk</a>
-          address:
-            line_1: Ministry of Justice
-            line_2: 102 Petty France
-            line_3: London
-            post_code: SW1H 9AJ
-      contact_or_complaint:
-        header: Contact us or make a complaint
-        data_protection_office:
-          list_heading_html: "Contact the <abbr title='Ministry of Justice'>MOJ</abbr> Data Protection Officer if you:"
-          list: |
-            have a question about anything in this privacy notice
-            think that your personal data has been misused or
-          contact_details:
-            email_address_html: <a href="mailto:casework@ico.org.uk">casework@ico.org.uk</a>
-            telephone: "Telephone: 0303 123 1113"
-            text_phone: "Textphone: 01625 545860"
-            call_hours: "Monday to Friday, 9am to 4:30pm"
-            call_costs_html: <a href="https://www.gov.uk/call-charges">Find out about call charges</a>
-        commissioner_office:
-          list_heading_html: "Contact the Information Commissioner's Office (<abbr title='Information Commissioner's Office'>ICO</abbr>):"
-          list: |
-            to make a complaint
-            for independent advice about data protection
-          address:
-            title: Information Commissioner's Office
-            line_1: Wycliffe House
-            line_2: Water Lane
-            line_3: Wilmslow
-            line_4: Cheshire
-            post_code: SK9 5AF
+          line_1: Ministry of Justice
+          line_2: 3rd Floor, Post Point 3.20
+          line_3: 10 South Colonnades
+          line_4: Canary Wharf
+          line_5: London
+          post_code: E14 4PU
+        email_address_html: <a href="mailto:privacy@justice.gov.uk">privacy@justice.gov.uk</a>
+        info_2: For more information on how and why your information is processed, please see the information provided when you accessed our services or were contacted by us.
+      complaints:
+        header: Complaints
+        info_1: "When we ask you for information, we will comply with the law. If you consider that your information has been handled incorrectly, you can contact the Information Commissioner for independent advice about data protection. You can contact the Information Commissioner at:"
+        address:
+          title: Information Commissioner's Office
+          line_1: Wycliffe House
+          line_2: Water Lane
+          line_3: Wilmslow
+          line_4: Cheshire
+          post_code: SK9 5AF
+          telephone: "Tel: 0303 123 1113"
+        link_html: <a href='www.ico.org.uk'>www.ico.org.uk</a>
```

Comment:
This may need the `http` prefix, the actual link is rendering strangely ```https://ap-2197-update-privacy-notice-applyforlegalaid-uat.apps.live-1.cloud-platform.service.justice.gov.uk/www.ico.org.uk```

Reply:
good spot!